packages/kolibri-tools/package.json

```diff
@@ -101,9 +101,6 @@
     "node": "10.x",
     "npm": ">= 3"
   },
-  "optionalDependencies": {
-    "kolibri": "0.13.0-dev.10"
```

Comment:
I think given that we want to get rid of this package, this is probably OK - but just being aware that outside of the `packages` workspace context, I can imagine this being an issue, as there's no indication that this should be installed.

Reply:
why is it optional?

Reply:
Because it's only needed for running tests, not for linting or building plugins.